logs_pipeline_list.go

```diff
@@ -0,0 +1,29 @@
+package datadog
+
+const (
+	logsPipelineListPath = "/v1/logs/config/pipeline-order"
+)
```

Comment:
Structurally it's good, but I would emphasise on the sorted aspect of the pipeline ids in the method names